pkg/plugin/constants.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
I added this file because this constant is actually shared with both aggregation and driver subpackages.